gluefactory/models/base_model.py

```diff
@@ -125,3 +128,31 @@ def _forward(self, data):
     def loss(self, pred, data):
         """To be implemented by the child class."""
         raise NotImplementedError
+
+    def load_state_dict(self, state_dict: Mapping[str, Any], strict: bool = True):
+        """Load the state dict of the model, and set the model to initialized."""
+        incompatible_keys = super().load_state_dict(state_dict, strict=strict)
+        self.set_initialized(True)
+        return incompatible_keys
```

Comment:
```suggestion
    def load_state_dict(self, *args, **kwargs):
        """Load the state dict of the model, and set the model to initialized."""
        ret = super().load_state_dict(*args, **kwargs)
        self.set_initialized()
        return ret
```
For forward compatibility